requirements.txt

```diff
@@ -1,2 +1,6 @@
-pytest==6.2.5
-flake8==4.0.1
\ No newline at end of file
+flake8==5.0.4
+flake8-annotations==2.9.1
+flake8-quotes==3.3.1
+flake8-variables-names==0.0.5
+pep8-naming==0.13.2
+pytest==7.1.3
```

Comment:
Exclude the content of the `__pycache__` directory and the directory itself from the PR.